### PR TITLE
Add a UI to catch email

### PR DESCRIPTION
**Pull request by simfeld**

### CODE REVIEW

File: backend/config/autoload/mail.global.dev.php [hunk 1]
```diff
@@ -3,9 +3,10 @@
 return [
     'laminas_mail' => [
         'transport' => [
-            'type' => 'file',
+            'type' => 'smtp',
```
Comment (simfeld) [inline backend/config/autoload/mail.global.dev.php:6]: @pmattmann I didn't find out what this option does/whether it is necessary

File: docker-compose.yml [hunk 2]
```diff
@@ -127,6 +127,12 @@ services:
     environment:
       - CORS=true
 
+  mail:
+    image: mailhog/mailhog
+    container_name: 'ecamp3-mail'
+    ports:
+      - '8025:8025' # web UI
```
Comment (simfeld) [inline docker-compose.yml:134]: @carlobeltrame would it be possible to include this in the dev deployment and expose this port, for example as `mail-dev.ecamp3.ch`?

**Comment by simfeld** [comment]
Looks like this

grafik

**Comment by BacLuc** [comment]
Does this allow to fetch the mails in a cypress test?

**Review by BacLuc — APPROVED** [review]

**Comment by carlobeltrame** [comment]
> Does this allow to fetch the mails in a cypress test?

I know [mailcatcher](https://mailcatcher.me/) allows this, not sure about mailhog.

File: backend/config/autoload/mail.global.dev.php (hunk 1 above)
Comment (carlobeltrame) [reply backend/config/autoload/mail.global.dev.php:6]: It is used to configure what type of mail transfer to use and where the SMTP server lies. See my attempt of the same here: https://github.com/carlobeltrame/ecamp3/commit/9491608be8aa24537d81a653f502b9a255304283

Comment (simfeld) [reply backend/config/autoload/mail.global.dev.php:6]: Ok, I wasn't sure what to put there as `type` but `smtp` seems to work

**Comment by simfeld** [comment]
> > Does this allow to fetch the mails in a cypress test?
> 
> I know [mailcatcher](https://mailcatcher.me/) allows this, not sure about mailhog.

It supports an API allowing to search for messages, should be possible

grafik

**Comment by simfeld** [comment]
TODO:

- [x] New subdomain mail-dev.ecamp3.ch @pmattmann 
- [x] Set variable `DEVEL_MAIL_SERVER_URL` in GitHub secrets

File: docker-compose.yml (hunk 2 above)
Comment (pmattmann) [reply docker-compose.yml:134]: Können wir hier auch einen 3000-er Port verwenden?
So dass alle eCamp3-Dienste unter einem 300x Port laufen. Macht es etwas einfacher, sich die Ports zu merken.

**Review by carlobeltrame — APPROVED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (simfeld) [reply docker-compose.yml:134]: Klar, wir können lokal mappen was uns beliebt. Oder geht es dir auch um den Port im Container?

Comment (simfeld) [reply docker-compose.yml:134]: #1419

**Pull request merged**